src/components/Main/Results/DeterministicLinePlot.tsx

```diff
@@ -362,7 +362,7 @@ export function DeterministicLinePlot({
                   />
                 ))}
 
-                {translatePlots(t, areasToPlot).map((d) => (
```

Comment:
Translation has already been performed, so this function is redundant and can be removed.